plugin-build/build.gradle.kts

```diff
@@ -70,3 +70,34 @@ if (gradle.gradleVersion >= "6.6.0") {
     val publish = extensions.getByType(MavenPublishPluginExtension::class.java)
     publish.releaseSigningEnabled = BuildUtils.shouldSignArtifacts()
 }
+
+subprojects {
```

Comment:
as this is a module and not a high level build script, you need to remove the `subprojects` and keep all the rest